grammars/php.cson

```diff
@@ -219,7 +223,17 @@
   'constants':
     'patterns': [
       {
-        'begin': '(?xi)(?=\n\t\t\t            (\n\t\t\t                (\\\\[a-z_][a-z_0-9]*\\\\[a-z_][a-z_0-9\\\\]*)|\n\t\t\t                ([a-z_][a-z_0-9]*\\\\[a-z_][a-z_0-9\\\\]*)\n\t\t\t            )\n\t\t\t            [^a-z_0-9\\\\])'
+        'begin': '''
+          (?xi)
+          (?=
+            (
+              (\\\\[a-z_][a-z_0-9]*\\\\[a-z_][a-z_0-9\\\\]*)
```

Comment:
Possible, but take a lot of time and mostly will be pointless. Itself capturing group is explained in captures section as of parts for regex it possible to leave comments like "namespace" but it simply not improve regex understanding.